jenkins/scripts/integration_test.sh

```diff
@@ -179,33 +180,64 @@ fi
 
 cat "${CI_DIR}/integration_test_env.sh" >>"${CI_DIR}/files/${TEMP_FILE_NAME}"
 
-# Send Remote script to Executer
-scp \
-  -o StrictHostKeyChecking=no \
-  -o UserKnownHostsFile=/dev/null \
-  -i "${METAL3_CI_USER_KEY}" \
-  "${CI_DIR}/files/run_integration_tests.sh" \
-  "${METAL3_CI_USER}@${TEST_EXECUTER_IP}:/tmp" >/dev/null
-
-scp \
-  -o StrictHostKeyChecking=no \
-  -o UserKnownHostsFile=/dev/null \
-  -i "${METAL3_CI_USER_KEY}" \
-  "${CI_DIR}/files/${TEMP_FILE_NAME}" \
-  "${METAL3_CI_USER}@${TEST_EXECUTER_IP}:/tmp/vars.sh" >/dev/null
-
+if [[ -n "${SSH_JUMP_HOST}" ]]; then
+    # Send Remote script to Executer
+    scp \
+      -o StrictHostKeyChecking=no \
```

Comment:
Please use the `ssh_options` array here too, these `scp` are becoming unreadable and unmaintainable. We should construct the ssh_options, and not duplicate the actual `scp` lines at all.